ExampleMod/Content/Biomes/ExampleWaterStyle.cs

```diff
@@ -21,5 +23,13 @@ public class ExampleWaterStyle : ModWaterStyle
 
 		public override Color BiomeHairColor()
 			=> Color.White;
+
+		public override byte GetRainType() {
+			return (byte)Main.rand.Next(3);
+		}
+
+		public override Texture2D GetRainTexture() {
+			return Request<Texture2D>("ExperimentMod/Content/Biomes/ExampleRain").Value;
+		}
```

Comment:
Should be "ExampleMod/..."

Reply:
Also get rid of the `using static Terraria.ModLoader.ModContent;` and fix the errors that come from it by using ModContent class directly. static usings are discouraged for ExampleMod

Reply:
Cool so it looks like that entire file wasnt yet updated to ExampleMod standards. The `=>` notation for method bodies is not used in it. Revert it to regular block bodies.